src/IbuMethods.cpp

```diff
@@ -61,3 +67,49 @@ double IbuMethods::rager(double AArating, double hops_grams, double finalVolume_
 
    return (hops_grams*utilization*AArating*1000)/(finalVolume_liters*(1+gravityFactor));
 }
+
+
+/*!
+ * \author Daniel Pettersson (pettson81@gmail.com)
+ *
+ * \brief Calculates the IBU by Greg Noonans formula
+ */
+double IbuMethods::noonan(double AArating, double hops_grams, double finalVolume_liters, double wort_grav, double minutes)
+{
+    double gramsPerOz = 28.3495231;
+    double litersPerFiveGallons = 18.9270589; // US gallons
+    double volumeFactor = litersPerFiveGallons / finalVolume_liters;
+    double hopsFactor = hops_grams/ gramsPerOz;
+
+    //using 60 minutes as a general table
```

Comment:
I have only found tables for 30 and 60 minutes. These are the same up to 1.050 (1/1).
1.050 - 1.065 (0.9333/0.9286)
1.065-1.085 (0,8667/0,8214)
and above 1.085 (0,8/0,75).